src/components/CodeEditor/Editor.tsx

```diff
@@ -72,12 +102,22 @@ class Editor extends Component<EditorProps> {
 			});
 
 			monaco.editor.setTheme('myCustomTheme');
+			editor.setModel(model);
+			model.onDidChangeContent(() => {
+				this.onValueChange(model.getValue());
+			});
 			await liftOff(monaco);
 		}
 	};
 
 	render() {
-		return <div style={{ height: '70vh' }} ref={this.editorRef} />;
+		return (
+			<div>
+				<div style={{ height: '70vh' }} ref={this.editorRef} />
+				<br />
+				<Button onClick={this.onSave}>Save File</Button>
```

Comment:
How about adding a shortcut instead of a button ?

Reply:
Here is the shortcut snippet: https://microsoft.github.io/monaco-editor/playground.html#interacting-with-the-editor-adding-an-action-to-an-editor-instance 

We can do using `CMD + S`. Wdyt ?

Reply:
For sure, but in my opinion, the button also has to be there as it is better user experience if some is not able to figure out a shortcut.

Reply:
I think we can keep the button in the mobile view but the desktop, shortcut makes more sense. Consider the Local Editor that we have it doesn't come with a save button. Also, we may have circle on top navbar for unsaved file and we can show tooltip there.